### DISCUSSION TITLE
Better Grafana dashboard

### DISCUSSION THREAD

**Discussion by libook** [sladkoff/minecraft-prometheus-exporter#184] [General]
Hi, @sladkoff .
I spent two days redoing two dashboard.
If you like it, consider putting it in your dashboard example.
https://gist.github.com/libook/f4bdf00f7338d388d061699dbb0931da

**Comment by sladkoff** [comment]
Hi @libook !

Nice work and thanks for mentioning it.

Feel free to create a PR with the new dashboards. I'd suggest you add them to a new subfolder below https://github.com/sladkoff/minecraft-prometheus-exporter/tree/master/dashboards.

Would be nice if you could also add a screenshot of each dashboard to that directory.

Thanks!

**Comment by libook** [comment]
Please view:
https://github.com/sladkoff/minecraft-prometheus-exporter/pull/187
